class.jetpack.php

```diff
@@ -2822,7 +2822,7 @@ function throw_error_on_activate_plugin( $plugin ) {
 			}
 
 			if ( $throw ) {
-				trigger_error( sprintf( __( 'Jetpack contains the most recent version of the old &#8220;%1$s&#8221; plugin.', 'jetpack' ), 'WordPress.com Stats' ), E_USER_ERROR );
+				trigger_error( sprintf( __( 'Jetpack contains the most recent version of the old &#8220;%1$s&#8221; plugin.', 'jetpack' ), 'Site Stats' ), E_USER_ERROR );
```

Comment:
Fixed in https://github.com/Automattic/jetpack/commit/fd650dfc0515ecbc22965c939c95042b87042105